packages/x-components/src/components/index.ts

```diff
@@ -30,10 +30,5 @@ export { default as SnippetCallbacks } from './snippet-callbacks.vue';
 export { default as PageLoaderButton } from './page-loader-button.vue';
 
 // Utils
-export * from './decorators/bus.decorators';
-export * from './decorators/debounce.decorators';
-export * from './decorators/injection.consts';
-export * from './decorators/injection.decorators';
-export * from './decorators/store.decorators';
```

Comment:
I'd also remove the files so we won't forget them

Reply:
We are still using `injection.consts` across the project. Also, the const uses other types of `injection.decorators` We should do it in a separate PR, WDYT?

Reply:
You are so right! then I think we can even keep exporting the injection.const for now. Probably we will not remove this file after all.

Reply:
We could also remove more things in this PR:
- `dynamicPropsMixin`
- `xComponentMixin`
WDYT?